File: python/final_value_of_variable_after_performing_operations_2011.py
class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        out = 0
        for op in operations:
            match op:
                case "++X":
                    out+=1
                case "X++":
                    out +=1
                case "--X":
                    out-=1
                case "X--":
                    out-=1
        return out

File: rust/src/final_value_of_variable_after_performing_operations_2011.rs
use crate::Solution;
impl Solution {
    pub fn final_value_after_operations(operations: Vec<String>) -> i32 {
        let mut out = 0;
        for op in operations {
            match op.as_str() {
                "++X" | "X++" => {
                    out +=1;
                },
                "--X" | "X--" => {
                    out -= 1;
                },
                _ => {

                }

            }
        }
        out
    }
}
